.github/workflows/ci.yml

```diff
@@ -16,6 +16,21 @@ jobs:
       - run: make apiv2
       - run: git diff --exit-code
 
+  build:
+    name: Build Alertmanager for common architectures
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        thread: [ 0, 1, 2 ]
```

Comment:
This complexity is not needed for alertmanager

Reply:
I took this from the[ circleci build job](https://github.com/prometheus/alertmanager/blob/c6be0bcabf37e837d24786e3d5c10565ac499531/.circleci/config.yml#L127-L135) thinking its a reasonable check before merge. Happy to drop if it isn't needed.

Reply:
Also https://github.com/prometheus/alertmanager/pull/3530 wants to extend this.